### PR TITLE
Don't duplicate code.

### PR DESCRIPTION
**Pull request by paulmillr**
-15 LOC.

### CODE REVIEW

File: lib/chars.js [hunk 1]
```diff
@@ -65,17 +62,6 @@ chars.ESC_TEMP = {
...
-  '__TEMP_EXCL__'   : '!',
-  '__TEMP_STAR__'   : '*',
-  '__TEMP_PLUS__'   : '+',
-  '__TEMP_COMMA__'  : ',',
-  '__TEMP_LTPAREN__': '(',
-  '__TEMP_RTPAREN__': ')',
-  '__TEMP_LTBRACK__': '[',
-  '__TEMP_RTBRACK__': ']',
-};
+chars.TEMP = reverse(chars.TEMP);
```
Comment (es128) [inline lib/chars.js]: Didn't you mean `chars.ESC_TEMP` here?

Comment (paulmillr) [inline lib/chars.js]: uhh

**Comment by es128** [comment]
LGTM

**Comment by jonschlinkert** [comment]
lgtm too, thanks

**Pull request merged**